src/components/Login.js

```diff
@@ -3,7 +3,44 @@ import React, { useEffect, useState } from "react";
 function Login() {
     const [statesHere, setStateshere] = useState("");
 
-    return <div>OUR HEADER GOES HERE</div>;
+    const submitHandler = event => {
+        event.preventDefault();
+    };
+    const onInputChange = event => {
+        // console.log`name is currently ${nameInput.value} email is currently ${emailInput.value} role is currently${roleInput.value}`();
+
+        setNewMember({
+            // ...newMember,
```

Comment:
This is right on track. Changes to be made:
1. Uncomment lines 13-14
2. Create some state called `newMember` that's a new user object. Since you have inputs for a name and password, you can initialize `newMember` with the keys `name` and `password` set to empty strings
3. The inputs need `value` and `onChange` props so they will update their respective parts of the `newMember` state. The name input is there, you just need to uncomment lines 27-28. And then the password input should be of `type = "password"` and give it a value prop, too